src/applications/appeals/onramp/utilities/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export const printErrorMessage = message =>
+  // eslint-disable-next-line no-console
+  console.error(message);
+
+/**
+ * Used for summary screens
+ * Takes a list of 0 or more items and returns the proper markup
+ *
+ * @param {*} items List of 0 or more items to render
+ * @param {*} includeAnds Flag indicating whether to include ', and'
+ * after each list item except the last one
+ * @param {*} paragraphClass Optional param for class name(s) for the single item
+ * @param {*} listItemClasses Optional param for class name(s) for each list item
+ */
+export const renderSingleOrList = (
+  items,
+  includeAnds,
+  paragraphClass = null,
+  listItemClasses = null,
+  testId = null,
+) => {
+  if (!items?.length) return null;
+
+  if (items.length === 1) {
+    return (
+      <p className={paragraphClass || ''} data-testid={`${testId}-0`}>
+        {items[0]}
+        {includeAnds && '.'}
```

Comment:
Nitpick, but since `includeAnds` is controlling more than just adding "and", could we rename it to be more descriptive (maybe `addPunctuation` or `useSentenceFormat`)?